### PR TITLE
Documentation/profiles

### PR DESCRIPTION
**Pull request by lmontopo**

### CODE REVIEW

File: source/includes/profiles.md [hunk 1]
```diff
@@ -1 +1,44 @@
...
+Authorization: Token "YOUR SDE ACCESS TOKEN"
+```
+
+```http
+HTTP/1.1 200 OK
+Content-Type: application/json
+
+{
+    "results": [
+    {
```
Comment (sibartlett) [inline source/includes/profiles.md]: The items in the array should be indented by 4 more spaces.

File: source/includes/profiles.md [hunk 2]
```diff
@@ -1 +1,44 @@
...
+        "id": "P3",
+        "name": ".NET Web App",
+        "description": "Microsoft .NET Web Application",
+        "answers":["A1061", "A740", "A48", "A4", "A2"],
+        "logo_url": "/static/images/logo-net.png"
+    }
+    ]
+}
+```
+This endpoint retrieves a list of Application resources.
```
Comment (funkaoshi) [inline source/includes/profiles.md]: "... list of Profile resources."

File: source/includes/profiles.md [hunk 3]
```diff
@@ -1 +1,44 @@
...
+
+### Query Parameters
+
+The following parameters may be used to filter the application resources in the response.
+
+
+Parameter | Description
+----------|-------------
+ordering |  Returns a list of application resources ordered by the specified field. This endpoint supports ordering by id, name and description.
```
Comment (funkaoshi) [inline source/includes/profiles.md]: Is it OCD to suggest we line these things up?

```
Parameter | Description
----------|-------------
ordering  | Returns a list of application resources ordered by the specified field. This endpoint supports ordering by id, name and description.
```

Comment (lmontopo) [inline source/includes/profiles.md]: Not OCD.  Good idea, I'll do that now.

File: source/includes/profiles.md (hunk 2 above)
Comment (lmontopo) [inline source/includes/profiles.md]: Thanks!

File: source/includes/profiles.md (hunk 1 above)
Comment (lmontopo) [inline source/includes/profiles.md]: right, thanks!

**Pull request merged**